service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -944,5 +955,599 @@
+    "targetableText":"Is there even a perfume strong enough to cover [TARGET]'s stench?",
+    "targetableSsml":"Is there even a perfume strong enough to cover [TARGET]'s stench?"
+  },
+  {
+    "quipGroup":"FEEL_BETTER_ABOUT_LIFE",
+    "text":"If I ever need to feel better about my life, all I have to do is look at yours.",
+    "ssml":"If I ever need to feel better about my life, all I have to do is look at yours.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"If I ever need to feel better about my life, all I have to do is look at [TARGET]'s.",
+    "targetableSsml":"If I ever need to feel better about my life, all I have to do is look at [TARGET]'s."
+  },
+  {
+    "quipGroup":"ENJOY_END_OF_CONVERSATION",
+    "text":"The parts of our conversations I enjoy the most, are when they end.",
+    "ssml":"The parts of our conversations I enjoy the most, are when they end.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"The parts of my conversations with [TARGET] that I enjoy the most, are when they end.",
+    "targetableSsml":"The parts of my conversations with [TARGET] that I enjoy the most, are when they end."
+  },
+  {
+    "quipGroup":"BRING_SILENCE_TO_CONVERSATION",
+    "text":"The greatest gift you could bring to a conversation would be silence.",
+    "ssml":"The greatest gift you could bring to a conversation would be silence.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"The greatest gift [TARGET] could bring to a conversation would be silence.",
+    "targetableSsml":"The greatest gift [TARGET] could bring to a conversation would be silence."
+  },
+  {
+    "quipGroup":"SPINELESS",
+    "text":"You're so spineless you could be classified as an invertebrate.",
+    "ssml":"You're so spineless you could be classified as an invertebrate.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"BLAND_PERSONALITY",
+    "text":"Your personality is so bland we could feed it to a vegan.",
+    "ssml":"Your personality is so bland we could feed it to a vegan.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"[TARGET]'s personality is so bland we could feed it to a vegan.",
+    "targetableSsml":"[TARGET]'s personality is so bland we could feed it to a vegan."
+  },
+  {
+    "quipGroup":"YOUR_LIFE_DOESNT_SUCK",
+    "text":"Your life doesn't suck, you do.",
+    "ssml":"Your life doesn't suck, you do.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"DUMB_DECISIONS",
+    "text":"You don't have bad luck. You just make dumb decisions.",
+    "ssml":"You don't have bad luck. You just make dumb decisions.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"COMA_PHONE_CHARGE",
```

Comment:
Possibly. I'm not sure how to lighten the phrasing while maintaining the theme though. We can probably come up with something. Or a variation on the theme that doesn't involve murder.